plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -248,7 +255,7 @@ func ClusterRoles() []rbac.ClusterRole {
 		{
 			// a role for nodes to use to have the access they need for running pods
 			ObjectMeta: metav1.ObjectMeta{Name: "system:node"},
-			Rules:      NodeRules(),
+			Rules:      addPodSecurityPolicy(pspEnabled, "privileged", NodeRules()...),
```

Comment:
I wouldn't include that in the node role, I'd keep PSP permissions in distinct roles. I also wouldn't mix layers... the `privileged` PSP name is defined in an add-on, I would define the role and binding for that in the add-on as well, not in a bootstrap role.